.travis.yml

```diff
@@ -3,6 +3,8 @@ node_js: 10
 env:
   - NODE_ENV=ci
   - NODE_ENV=ci-betanet
+    #  - NODE_ENV=devnet
+  - NODE_ENV=betanet
```

Comment:
Not sure whether this is a good idea. If betanet is down, near-api-js development would be blocked.

Reply:
Agree with Bowen here, I think it's too early have to have CI check betanet. In time, perhaps